lib/Dialect/AIEVec/TransformOps/AIEVecTransformOps.cpp

```diff
@@ -239,7 +231,7 @@ DiagnosedSilenceableFailure transform::VectorizeContractionOp::applyToOne(
   for (unsigned i = 0; i < numOuterMostDims; i++)
     remOuterDims.push_back(getAffineDimExpr(i, ctx));
   unsigned numResults = indexingMaps[0].getNumResults();
-  llvm::ArrayRef<int64_t> positions = {numResults - 2, numResults - 1};
+  SmallVector<int64_t> positions = {numResults - 2, numResults - 1};
```

Comment:
Sorry this bit you :(

Reply:
It's all good, you solved it yourself so... 😄